aiohttp/web.py

```diff
@@ -197,6 +197,23 @@ def add_subapp(self, prefix, subapp):
             subapp._set_loop(self._loop)
         return resource
 
+    def add_subdomain(self, subdomain, subapp):
```

Comment:
Can we rename this to add_domain(self, domain, subapp)? The way you did it actually enables multi-site apps on different domains, not only subdomains I think, so why don't just change the name?

Reply:
I agree.